config/post-install/v0.21.0/placeholder.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package postinstall is a placeholder that allows us to pull in config files
+// via go mod vendor.
+package postinstall
```

Comment:
@lionelvillard @pierDipi the PingSource migration is using `v0.21.1`, we should try to merge this with PRs like #4658 to make sure we have single post install directory. Do you think we should bring up in the eventing meeting?

Reply:
Do we remove the post-install directory after 0.20.0 is cut add this commit back in?

Reply:
> Do we remove the post-install directory after 0.20.0 is cut add this commit back in?

yes, like serving and as expected by the operator (@houshengbo, @Cynocracy)